recipes/autometa/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "Autometa" %}
+{% set sha256 = "b9b82c15e6f4d76c49d3e80d38a901606e7fd8ba945f83657535d27bb3d8860b" %}
+{% set version = "2.0.0a0" %}
```

Comment:
We try not to package alpha releases. If you want this in bioconda please tag an proper release version.

Reply:
This is my first time contributing to bioconda, so I have been going through this as a sort of trial to see if I can get this to work. When we release a production ready version, I will update. Should the PR be closed until then? We currently do not have a production ready version available.

Reply:
Yeah, we might as well close it until then. Definitely keep your fork though, since it looks like a minor update should get this ready for distribution once your release is tagged :)